molo/commenting/models.py

```diff
@@ -19,3 +21,9 @@ class MPTTMeta:
     class Meta:
         app_label = 'commenting'
         ordering = ['-submit_date', 'tree_id', 'lft']
+        
+@receiver(comment_was_flagged, sender=MoloComment)
+def remove_comment_if_flag_limit(sender, comment, flag, created, **kwargs):
+    if (comment.flags.count() >= 3):
```

Comment:
@miltontony I don't think a threshold flag in the settings will be the correct way to do this (due to the aforementioned requirement that it be editable) the django docs state that you should never change settings during runtime
https://docs.djangoproject.com/en/1.8/topics/settings/#altering-settings-at-runtime